QuizApp/Common/CoreUI.swift

```diff
@@ -0,0 +1,124 @@
+import Foundation
+import UIKit
+
+class CoreUI {
```

Comment:
None of these need to be a part of a class. It's enough to put them in a folder but have a standalone classes (like `class RoundedTextInput` in its own file)